bug: Error: Cannot find module 'sharp'

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Performed `npm install -g appium@2.0.0-rc.1` on node 18.16.0 installed using nvm.
Appium installs successfully but shows below error after running `appium`

```
Error: Cannot find module 'sharp'

Require stack:

- /Users/REMOVED/code/.nvm/versions/node/v18.16.0/lib/node_modules/appium/node_modules/@appium/support/build/lib/mjpeg.js

- /Users/REMOVED/code/.nvm/versions/node/v18.16.0/lib/node_modules/appium/node_modules/@appium/support/build/lib/index.js

- /Users/REMOVED/code/.nvm/versions/node/v18.16.0/lib/node_modules/appium/node_modules/@appium/support/index.js

- /Users/REMOVED/code/.nvm/versions/node/v18.16.0/lib/node_modules/appium/build/lib/logsink.js

- /Users/REMOVED/code/.nvm/versions/node/v18.16.0/lib/node_modules/appium/build/lib/main.js

- /Users/REMOVED/code/.nvm/versions/node/v18.16.0/lib/node_modules/appium/index.js

    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)

    at Function.Module._load (node:internal/modules/cjs/loader:920:27)

    at Module.require (node:internal/modules/cjs/loader:1141:19)

    at require (node:internal/modules/cjs/helpers:110:18)

    at Object.<anonymous> (/Users/REMOVED/code/.nvm/versions/node/v18.16.0/lib/node_modules/appium/node_modules/@appium/support/lib/mjpeg.js:4:1)

    at Module._compile (node:internal/modules/cjs/loader:1254:14)

    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1308:10)

    at Module.load (node:internal/modules/cjs/loader:1117:32)

    at Function.Module._load (node:internal/modules/cjs/loader:958:12)

    at Module.require (node:internal/modules/cjs/loader:1141:19)

    at require (node:internal/modules/cjs/helpers:110:18)

    at Object.<anonymous> (/Users/REMOVED/code/.nvm/versions/node/v18.16.0/lib/node_modules/appium/node_modules/@appium/support/lib/index.js:12:1)

    at Module._compile (node:internal/modules/cjs/loader:1254:14)

    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1308:10)

    at Module.load (node:internal/modules/cjs/loader:1117:32)

    at Function.Module._load (node:internal/modules/cjs/loader:958:12)
```

### Expected Behavior

Appium server launches normally.

### Minimal Reproducible Example

`npm install -g appium@2.0.0-rc.1`
Then run `appium`

### Environment

- Operating system: MacOS Monterey
- If running via `appium` CLI...
  - Appium CLI version (output of `appium --version`): appium@2.0.0-rc.1
  - Node.js version (output of `node --version`): 18.16.0
  - `npm` version (output of `npm --version`): 9.5.1
  - Last `appium` version which did _not_ exhibit the problem:
- If running Appium Desktop, its version:
- Appium driver(s) and their version(s):
- Appium plugin(s) and their version(s):
- Platform and version under test:
- Real device or emulator/simulator:


### Link to Appium Logs

_No response_

### Futher Information

Potential PR of interest (?) : https://github.com/appium/appium/pull/18733
We were unable to install appium 2 previously because of the sharp dependency, now it installs great but it does not launch.

--- a/packages/support/lib/image-util.js
+++ b/packages/support/lib/image-util.js
@@ -3,7 +3,7 @@ let _sharp;
 /**
  * @returns {import('sharp')}
  */
-function requireSharp() {
+export function requireSharp() {
   if (!_sharp) {
     try {
       _sharp = require('sharp');

--- a/packages/support/lib/mjpeg.js
+++ b/packages/support/lib/mjpeg.js
@@ -1,7 +1,7 @@
 import _ from 'lodash';
 import log from './logger';
 import B from 'bluebird';
-import sharp from 'sharp';
+import {requireSharp} from './image-util';
 import {Writable} from 'stream';
 import {requirePackage} from './node';
 import axios from 'axios';
@@ -77,7 +77,7 @@ class MJpegStream extends Writable {
     }
 
     try {
-      return await sharp(lastChunk).png().toBuffer();
+      return await requireSharp()(lastChunk).png().toBuffer();
     } catch (e) {
       return null;
     }

